tests/dialects/test_prql.py

```diff
@@ -5,56 +5,56 @@ class TestPRQL(Validator):
     dialect = "prql"
 
     def test_prql(self):
-        self.validate_identity("FROM x", "SELECT * FROM x")
```

Comment:
why did you change these tests?

Reply:
> Btw, it also seems that PRQL's keywords are case-sensitive. So `FROM x` is not valid, but `from x` is. You can use their [playground](https://prql-lang.org/playground/) environment to check these things:
> 
> <img alt="Screenshot 2024-04-11 at 1 39 35 PM" width="287" src="https://private-user-images.githubusercontent.com/46752250/321599622-a9e469e1-5108-473e-9e93-5a9230943367.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTI4NDk1MDMsIm5iZiI6MTcxMjg0OTIwMywicGF0aCI6Ii80Njc1MjI1MC8zMjE1OTk2MjItYTllNDY5ZTEtNTEwOC00NzNlLTllOTMtNWE5MjMwOTQzMzY3LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA0MTElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNDExVDE1MjY0M1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTQ1OGUzMTA2ZmQzN2Y4OWVmOWQ4NzMzNTQ3ZjkzMDUzMmE1OTM5YTQ2NjdjOTMxMmE3YzY3NmNmYzdlZmIyZGEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.G3dpm00529jqS7geU7gzjEHNewYaDO-34dZaxPGKzxg">

PRQL keyword are case sensitive so I just lowercase all of them